packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.js

```diff
@@ -415,7 +420,7 @@ function userPoolProviders(oAuthProviders, coreAnswers, prevAnswers) {
   if (answers.hostedUI) {
     res.hostedUIProviderMeta = JSON.stringify(
       oAuthProviders.map(el => {
-        const delimmiter = el === 'Facebook' ? ',' : ' ';
+        const delimmiter = ['Facebook', 'SignInWithApple'].includes(el) ? ',' : ' ';
```

Comment:
Can you write this as two strict equality comparisons instead of creating a new array and calling `includes()` for each iteration.